src/include/api_inline.h

```diff
@@ -0,0 +1,53 @@
+/*-
+ * Copyright (c) 2014-present MongoDB, Inc.
+ * Copyright (c) 2008-2014 WiredTiger, Inc.
+ *  All rights reserved.
+ *
+ * See the file LICENSE for redistribution information.
+ */
+
+#pragma once
+
+/*
+ * __wt_api_track_cursor_start --
+ *     Start tracking a cursor API entry point for statistics.
+ */
+static WT_INLINE void
+__wt_api_track_cursor_start(WT_SESSION_IMPL *session)
```

Comment:
Minor point on naming - begin/end and start/finish feel like more natural "name pairs" than start/end.